cpp/ps0000/main.cpp

```diff
@@ -1,16 +1,9 @@
-/******************************************************************************
+#include <iostream> // 2/2
 
-Welcome to GDB Online.
-GDB online is an online compiler and debugger tool for C, C++, Python, Java, PHP, Ruby, Perl,
-C#, OCaml, VB, Swift, Pascal, Fortran, Haskell, Objective-C, Assembly, HTML, CSS, JS, SQLite, Prolog.
-Code, Compile, Run and Debug online from anywhere in world.
-
-*******************************************************************************/
-#include <iostream>
-
-int main()
+int main() // 1/1
 {
-    std::cout<<"Hello World";
+    std::cout<<"Hello, World!"<<std::endl; // 2/4
 
-    return 0;
-}
\ No newline at end of file
+    return 0; // 1/1
+}
+// great work 😎 8/10 ⭐
```

Comment:
- [x] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 mark for your first commit